src/scripts/feed/PostEntry.js

```diff
@@ -0,0 +1,41 @@
+import { sendGif } from "./auth/provider.js"
+
+
+const mainContainer = document.querySelector("#giffygram")
+
+mainContainer.addEventListener("click", clickEvent => {
+    if(clickEvent.target.id === "submitGif") {
+
+        const usergifTitle = document.querySelector("input[name='gifTitle']").value
+        const usergifURL= document.querySelector("input[name='gifURL']").value
+        const usergifStory = document.querySelector("input[name='gifStory']").value
+
+        const dataToSendToAPI = {
```

Comment:
dataToSendToAPI needs a key-value for timestamp

I would suggest using this code to gen const for time

const timestamp = (new Date()).toLocaleDateString('en-US')